pkg/sched/intervals.go

```diff
@@ -314,6 +314,44 @@ func ScheduleIntervalSummary(items []Interval, policyTags *PolicyTags) string {
 	return summary
 }
 
+func ScheduleIntervalsPerDay(items []RetainInterval, policyTags *PolicyTags) uint32 {
```

Comment:
May be call this MaxPerDayInstances - since this is the max and not per day thing ? 
policyTags is not used ? can skip len(items) and do the range always

Reply:
Done